src/main/java/com/o19s/es/explore/StatisticsHelper.java

```diff
@@ -0,0 +1,87 @@
+/*
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.o19s.es.explore;
+
+import java.util.ArrayList;
+
+public class StatisticsHelper {
+    private final ArrayList<Float> data = new ArrayList<>(10);
+
+    private float min = Float.MAX_VALUE;
+    private float max = 0.0f;
+
+    StatisticsHelper() {
+
+    }
+
+    public void add(float val) {
+        data.add(val);
+
+        if(val < this.min) {
+            this.min = val;
+        }
+
+        if(val > this.max) {
+            this.max = val;
+        }
+    }
+
+    public float getMax() {
+        assert !data.isEmpty();
+
+        return max;
+    }
+
+    public float getMin() {
+        assert !data.isEmpty();
+
+        return min;
+    }
+
+    public float getMean() {
+        assert !data.isEmpty();
+
+        return getSum() / data.size();
+    }
+
+    public float getSum() {
+        assert !data.isEmpty();
+
+        float sum = 0.0f;
+
+        for(float a : data) {
+            sum += a;
```

Comment:
I have some concerns about overflows and precisions, some stats may be relatively large numbers and I worry that float is not well suited for this. Sadly float is the type used by lucene...
I don't have great solutions but this certainly something we'll have to address sooner or later, esp. if we want to combine features.